library/classes/Document.class.php

```diff
@@ -695,6 +845,48 @@ function createDocument(
         return '';
     }
 
+    /**
+     * Retrieves the document data that has been saved to the filesystem or couch db.  If the $decrypt flag is set to
+     * true, it will return the unencrypted version of the data for the document.
+     * @param bool $decrypt True if the document should be decrypted, false otherwise
+     * @throws BadMethodCallException Thrown if the method is called when the document has been marked as deleted or expired
+     * @return false|string Returns false if the data failed to decrypt, or a string if the data decrypts or is unencrypted.
+     */
+    function get_data($decrypt = false)
```

Comment:
The natural use when grabbing a document is that it is decrypted if that is the way it is stored. So, if have an override like this, would default it to the standard use case. maybe something like:
```
function get_data($force_no_decrypt = false)
```